manim/mobject/types/opengl_vectorized_mobject.py

```diff
@@ -322,6 +322,15 @@ def set_flat_stroke(self, flat_stroke=True, recurse=True):
     def get_flat_stroke(self):
         return self.flat_stroke
 
+    def set_rgba_array_direct(
+        self,
+        rgbas: np.ndarray,
+        name="stroke_rgba",
+        recurse=True,
+    ):
+        for mob in self.get_family(recurse):
+            mob.data[name] = rgbas.copy()
+
```

Comment:
I believe this method already exists in `opengl_mobject.py`, was this just a temp change or is there a reason for it?

Reply:
Yes I added it in another PR, but did this before it was merged so can remove it

Reply:
Removed the duplicate